### DISCUSSION TITLE
服务端数据库模式部署报错

### DISCUSSION THREAD

**Discussion by xwydq** [lobehub/lobehub#3172] [General Question] [answered]
$ bun run db:migrate
$ MIGRATION_DB=1 tsx scripts/migrateServerDB/index.ts
❌ Database migrate failed: ErrorEvent {
  [Symbol(kTarget)]: WebSocket {
    _events: [Object: null prototype] {
      error: [Function],
      message: [Function],
      close: [Function],
      open: [Function]
    },
    _eventsCount: 4,
    _maxListeners: undefined,
    _binaryType: 'arraybuffer',
    _closeCode: 1006,
    _closeFrameReceived: false,
    _closeFrameSent: false,
    _closeMessage: <Buffer >,
    _closeTimer: null,
    _errorEmitted: true,
    _extensions: {},
    _paused: false,
    _protocol: '',
    _readyState: 3,
    _receiver: null,
    _sender: null,
    _socket: null,
    _bufferedAmount: 0,
    _isServer: false,
    _redirects: 0,
    _autoPong: true,
    _url: 'wss://35.232.221.18/v2',
    _req: null,
    [Symbol(shapeMode)]: false,
    [Symbol(kCapture)]: false
  },
  [Symbol(kType)]: 'error',
  [Symbol(kError)]: Error: unable to verify the first certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1674:34)
      at TLSSocket.emit (node:events:518:28)
      at TLSSocket._finishInit (node:_tls_wrap:1085:8)
      at ssl.onhandshakedone (node:_tls_wrap:871:12) {
    code: 'UNABLE_TO_VERIFY_LEAF_SIGNATURE'
  },
  [Symbol(kMessage)]: 'unable to verify the first certificate'
}
error: script "db:migrate" exited with code 1
error: script "build-migrate-db" exited with code 1
error: script "postbuild" exited with code 1
Error: Command "bun run build" exited with 1

**Comment by arvinxx** [comment]
这个报错感觉是证书问题？

**Reply by xwydq** [reply]
DATABASE_URL=postgres://postgres:pwd@ip:5432/lobe?sslmode=require

是必须 要带 ?sslmode=require ？我不带的时候也是报错这个。我部署的 postgres没打开 ssl

**Reply by arvinxx** [reply]
爬下这个贴看看：https://github.com/lobehub/lobe-chat/discussions/3062

**Comment by RubuJam** [comment, marked as answer]
https://github.com/lobehub/lobe-chat/discussions/3062#discussioncomment-9894128
